src/components/LoginForm.vue

```diff
@@ -87,7 +104,11 @@ export default {
           this.$store.dispatch("login");
         }
       } catch (err) {
-        alert("Email or Password is wrong");
+        console.log("Catch Section");
+        this.error = true;
```

Comment:
@JatinThakur14 Instead of using error and the error msg separately, you can just use the string variable for both.